### PR TITLE
optimiz: master ui will render data in order

### PR DESCRIPTION
**Pull request by ningfdx**
# What problem are we solving?
Master UI http://localhost:9333/ change the order every time I refresh the page.
That make me feel bad.

# How are we solving the problem?
change the store var type from map to []struct, then do sort to keep order

### CODE REVIEW

File: weed/topology/data_center.go [hunk 1]
```diff
@@ -30,15 +31,24 @@ func (dc *DataCenter) GetOrCreateRack(rackName string) *Rack {
 	return rack
 }
 
-func (dc *DataCenter) ToMap() interface{} {
-	m := make(map[string]interface{})
-	m["Id"] = dc.Id()
-	var racks []interface{}
+type DataCenterMap struct {
```
Comment (chrislusf) [inline weed/topology/data_center.go]: how about calling this `DataCenterInfo`? 

Same for other `xxxMap` objects.

File: weed/topology/data_center.go [hunk 2]
```diff
@@ -30,15 +31,24 @@ func (dc *DataCenter) GetOrCreateRack(rackName string) *Rack {
...
+	Id    NodeId    `json:"Id"`
+	Racks []RackMap `json:"Racks"`
+}
+
+func (dc *DataCenter) ToMap() DataCenterMap {
```
Comment (chrislusf) [inline weed/topology/data_center.go]: Change `ToMap` to `ToInfo`?

Same for other places.

File: weed/topology/topology_map.go [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
+)
+
+type TopologyMap struct {
+	Max         int64           `json:"Max"`
+	Free        int64           `json:"Free"`
+	DataCenters []DataCenterMap `json:"DataCenters"`
+	Layouts     []interface{}   `json:"Layouts"`
+}
 
 func (t *Topology) ToMap() interface{} {
```
Comment (chrislusf) [inline weed/topology/topology_map.go]: Change the return object type?

Comment (ningfdx) [reply weed/topology/topology_map.go]: OK, fix in new commit.

File: weed/topology/data_center.go (hunk 2 above)
Comment (ningfdx) [reply weed/topology/data_center.go]: OK, fix in new commit.

File: weed/topology/data_center.go (hunk 1 above)
Comment (ningfdx) [reply weed/topology/data_center.go]: OK, fix in new commit.

**Pull request merged**